CMP1903_A1_2324/Game.cs

```diff
@@ -17,5 +17,68 @@ internal class Game
 
         //Methods
 
+        public (Die, Die, Die, int) StartRoll()
+        {
+            int result = 0;
+            
+            Die dieOne = new Die();
+            Die dieTwo = new Die();
+            Die dieThree = new Die();
+
+            int rollOne = dieOne.Roll();
+            int rollTwo = dieTwo.Roll();
+            int rollThree = dieThree.Roll();
+            
+            result = rollOne + rollTwo + rollThree;
+            Console.WriteLine($"\nDice 1: {rollOne}");
+            Console.WriteLine($"Dice 2: {rollTwo}");
+            Console.WriteLine($"Dice 3: {rollThree}");
+            Console.WriteLine($"Sum: {result}");
+
+            return (dieOne, dieTwo, dieThree, result);
+        }
+
+        public int ExtraRolls()
+        {
+            int result = 0;
+            Die dieToRoll = new Die();
+            int numberOfRolls = 0;
+
+            while (true)
+            { 
+                Console.WriteLine("\nNumber of Rolls to execute: ");
+                
+                try
+                {
+                    numberOfRolls = Int32.Parse(Console.ReadLine());
+
+                    if (numberOfRolls < 1)
+                    {
+                        throw new Exception("Number of rolls out of bounds.");
+                    }
+
+                    break;
+                }
+                catch (Exception e)
```

Comment:
Specify the type of exception, to avoid other errors being undetected and improving readability